godot-macros/src/docs.rs

```diff
@@ -149,9 +149,17 @@ fn xml_escape(value: String) -> String {
 /// Calls [`siphon_docs_from_attributes`] and converts the result to BBCode
 /// for Godot's consumption.
 fn make_docs_from_attributes(doc: &[Attribute]) -> Option<String> {
-    let doc = siphon_docs_from_attributes(doc)
+    let lines: Vec<String> = siphon_docs_from_attributes(doc).collect();
+
+    let doc = lines
+        .iter()
+        // Strips the line of the first leading space if it exists. This space
+        // is commonly used to separate the comment from the `///` that precedes
+        // it, but interferes with the formatting of the resulting Markdown.
```

Comment:
Can you elaborate on the "interferes with" part; aren't extra spaces in markdown generally ignored (except 2 at end)?

Also, feel free to have comments [a bit longer](https://godot-rust.github.io/book/contribute/conventions.html#formatting) :slightly_smiling_face: no need to go counting characters, just roughly.

Reply:
At some point it was giving odd results, and this fixed it. I just tried it without stripping the initial spaces and now there seems to be nothing wrong with the output. Might possibly be something that got refactored away. Pulled this code out.